shared/chat/emoji-picker/index.tsx

```diff
@@ -321,14 +335,32 @@ class EmojiPicker extends React.PureComponent<Props, State> {
     </Kb.Box2>
   )
 
+  private getEmojiWidthWithPadding = () => {
+    return emojiWidthWithPadding
+  }
+
+  _sections = new Array<Section>()
+  private getSectionHeaderHeight = (sectionIndex: number) => {
+    return this._sections[sectionIndex].key === 'not-found' ? notFoundHeight : 32
+  }
+
+  _emojisPerLine = 1
+  private renderItem = ({item}: {item: Row; index: number}) => this.getEmojiRow(item, this._emojisPerLine)
+
+  private renderSectionHeader = ({section}) => {
+    return section.key === 'not-found' ? this.makeNotFound() : this.getSectionHeader(section.title)
+  }
+
   render() {
     const {bookmarks, sections} = getSectionsAndBookmarks(
       this.props.width,
       this.props.topReacjis,
       this.props.hideFrequentEmoji,
       this.props.customEmojiGroups
     )
+    this._sections = sections
```

Comment:
hacky but i don't want to refactor this entire thing. sections definitely thrash but its fine cause the RowItems are memoed